search.js

```diff
@@ -119,73 +149,123 @@ async function doRequest(cursor) {
 
 }
 
-function drawTable(data) {
```

Comment:
Apologies for the rough diff. I refactored drawTable to be makeTable and take in an optional tableStyle so that it could be reused for outputting as a TSV.